Detector/DetDriftChamber/compact/det.xml

```diff
@@ -15,13 +17,15 @@
     <gdmlFile  ref="materials.xml"/>
   </includes>
 
+
   <define>
-    <constant name="world_size" value="2226*mm"/>
+    <constant name="world_size" value="30*m"/>
```

Comment:
Hi @ihepzhangyao ,
Do you want to use 30 m for the world again?
Tao

Reply:
Use small world size again.